runtime/src/main/java/org/corfudb/runtime/view/SealServersHelper.java

```diff
@@ -136,7 +139,9 @@ public static void waitForQuorum(CompletableFuture<Boolean>[] completableFutures
                 QuorumFuturesFactory.getQuorumFuture(Boolean::compareTo, completableFutures);
         try {
             success = quorumFuture.get();
-        } catch (ExecutionException | InterruptedException e) {
+        } catch (InterruptedException ie) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either re-interrupt this method or rethrow the "InterruptedException". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2142)